cql2pgjson-cli/src/main/java/com/indexdata/cql2pgcli/CQL2PGCLIMain.java

```diff
@@ -0,0 +1,122 @@
+package com.indexdata.cql2pgcli;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.cli.*;
+import org.json.JSONException;
+import org.z3950.zing.cql.cql2pgjson.CQL2PgJSON;
+import org.z3950.zing.cql.cql2pgjson.FieldException;
+import org.z3950.zing.cql.cql2pgjson.QueryValidationException;
+import org.z3950.zing.cql.cql2pgjson.SchemaException;
+import org.z3950.zing.cql.cql2pgjson.SqlSelect;
+import org.json.JSONObject;
+
+public class CQL2PGCLIMain { 
+  
+  public static void main( String[] args ) {
+    Options options = new Options();
+    
+    Option database = Option.builder("d")
+        .hasArg()
+        .required(true)
+        .desc("Postgres database name")
+        .build();
+    
+    Option field = Option.builder("f")
+        .hasArg()
+        .required(true)
+        .desc("Postgres field name")
+        .build();
```

Comment:
The default should be `jsonb`, that what RMB almost always uses: https://github.com/folio-org/raml-module-builder#postgresql-integration

Reply:
Added default.